### PR TITLE
added filters

### PR DESCRIPTION
**Pull request by taabishkathawala**

### CODE REVIEW

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.example.brewhaha_android.Controllers
```
Comment (prabogey) [inline Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt:1]: Are you going to add the hookups to this file later (ie. for the buttons and stuff)?

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/HomeActivity.kt [hunk 2]
```diff
@@ -36,7 +39,7 @@ import java.util.concurrent.TimeUnit
 class HomeActivity(private val api: BackendConnection = BackendConnection()) : AppCompatActivity() {
     var _logout_button : MaterialButton? = null
     var _filter_button : MaterialButton? = null
-    var _brewery_search : EditText? = null
+    var _brewery_search : TextInputEditText? = null
```
Comment (prabogey) [inline Android/app/src/main/java/com/example/brewhaha_android/Controllers/HomeActivity.kt:42]: Whats the reason for this?

File: Android/app/src/main/res/layout/activity_home.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```
Comment (prabogey) [inline Android/app/src/main/res/layout/activity_home.xml]: Whats the use of a coorindator layout here, why is this better than a constraint layout?

File: Android/app/src/main/res/layout/activity_home.xml [hunk 4]
```diff
@@ -42,73 +53,49 @@
...
-                    android:layout_gravity="center_horizontal"
-                        android:gravity="right"
-                        android:layout_marginEnd="40dp"
-                    android:padding="12dp"
+                    android:layout_gravity="center_vertical|end"
+                    android:layout_marginStart="10dp"
+                    android:padding="8dp"
                     android:text="Filters"
-                    android:textColor="@color/cardview_light_background"
+                    android:textColor="@color/cardBackgroun"
```
Comment (prabogey) [inline Android/app/src/main/res/layout/activity_home.xml]: cardBackground?

**Review by prabogey — CHANGES_REQUESTED** [review]

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/HomeActivity.kt (hunk 2 above)
Comment (taabishkathawala) [reply Android/app/src/main/java/com/example/brewhaha_android/Controllers/HomeActivity.kt:42]: Making it material

File: Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt (hunk 1 above)
Comment (taabishkathawala) [reply Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt:1]: Yeah

Comment (taabishkathawala) [reply Android/app/src/main/java/com/example/brewhaha_android/Controllers/BottomSheetFragment.kt:1]: Filtering functionality isn't working yet

File: Android/app/src/main/res/layout/activity_home.xml (hunk 3 above)
Comment (taabishkathawala) [reply Android/app/src/main/res/layout/activity_home.xml]: Had to use it for the bottom sheet, but now I dont think it matters

File: Android/app/src/main/res/layout/activity_home.xml (hunk 4 above)
Comment (taabishkathawala) [reply Android/app/src/main/res/layout/activity_home.xml]: Yeah I just made new color themes for the app.

Comment (prabogey) [reply Android/app/src/main/res/layout/activity_home.xml]: you spelt it wrong though.

**Pull request merged**